tests/helper.js

```diff
@@ -184,4 +185,29 @@
   };
 
 
+  exports.ReceiptValidator = function ReceiptValidator(opt) {
+    opt = utils.defaults(opt, {
+      response: {status: 'ok'},
+      onRequest: function() {},
+      verifyUrl: new RegExp(
+        'https://receiptcheck-payments-alt\\.allizom\\.org/verify/'),
```

Comment:
actually, I seem to recall there was a bug in sinon where it would only accept a regex. To be honest, the memory-pain was still so intense that I did not attempt to see if a plain string would work. I guess I can take a minute to try :)